code/datums/ai_laws.dm

```diff
@@ -209,26 +194,16 @@
 			continue
 
 		add_inherent_law(line)
-	if(!inherent.len) //Failsafe to prevent lawless AIs being created.
-		log_law("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
-		add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-		add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-		add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
-		WARNING("Invalid custom AI laws, check silicon_laws.txt")
-		return
 
 /* General ai_law functions */
 
 /datum/ai_laws/proc/set_laws_config()
 	var/list/law_ids = CONFIG_GET(keyed_list/random_laws)
 	switch(CONFIG_GET(number/default_laws))
 		if(0)
-			add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-			add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-			add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+			set_laws_lawset("/datum/ai_laws/default/asimov")
```

Comment:
Would you like the path changed to a define?

Reply:
Don't reference typepaths like this if you can help it, it won't show up as an issue if the path is removed